src/Client/V2/Service/EmsApiService.cs

```diff
@@ -55,6 +55,11 @@ public EmsApiService( EmsApiServiceConfiguration config, EmsApiServiceHttpClient
         /// </summary>
         public EmsSecurablesAccess EmsSecurables { get; set; }
 
+        /// <summary>
+        /// Access to the admin securables routes.
+        /// </summary>
+        public AdminEmsSecurablesAccess AdminEmsSecurables { get; set; }
```

Comment:
Looks like this guy needs to be added to `InitializeAccessProperties`, I think that's why the new tests are failing